x-pack/legacy/plugins/ml/server/lib/check_privileges/check_privileges.ts

```diff
@@ -49,12 +76,12 @@ export function privilegesProvider(callWithRequest: callWithRequestType, xpackMa
         setActionPrivileges(cluster, privileges);
       }
     }
-    return { privileges, upgradeInProgress };
+    return { privileges, upgradeInProgress, isPlatinumLicense, mlEnabledInSpace };
   }
   return { getPrivileges };
 }
 
-function setGettingPrivileges(
+function setPlatinumGettingPrivileges(
```

Comment:
Rename to `setFullGettingPrivilegese` if the license type enum is set back to `FULL`?